papas/display/src/Display.cpp

```diff
@@ -14,7 +14,7 @@ Display::Display(std::list<ViewPane::Projection> views) {
     views = {ViewPane::Projection::xy, ViewPane::Projection::yz, ViewPane::Projection::xz};
   }
   /// Creates viewpanes
-  for (auto view : views) {
+  for (auto& view : views) {
```

Comment:
In the end I decided not to handle the display stuff for this pull request as it is not really part of the library.